### PR TITLE
Update project base

### PR DESCRIPTION
**Pull request by pratclot**

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# TODO list
+
+- add version catalog / platform project
+- add code formatter
+- create a screen in Flutter
```
Comment (paulacr) [inline TODO.md]: let's remove this line, we didn't decide this yet

File: buildtools/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#
```
Comment (paulacr) [inline buildtools/.gitattributes:1]: I do not see the point of this, was added in your branch, what is the purpose of using this?

File: buildtools/plugin/src/main/kotlin/buildtools/AndroidAppPlugin.kt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/*
+ * This Kotlin source file was generated by the Gradle 'init' task.
```
Comment (paulacr) [inline buildtools/plugin/src/main/kotlin/buildtools/AndroidAppPlugin.kt]: This comment is not needed

**Review by paulacr — COMMENTED** [review]
Please take a look, I added some comments

File: buildtools/plugin/src/main/kotlin/buildtools/AndroidAppPlugin.kt (hunk 3 above)
Comment (pratclot) [reply buildtools/plugin/src/main/kotlin/buildtools/AndroidAppPlugin.kt]: d3b17f0b4ae40a3d80f78a76aa38776c2696595a

File: TODO.md (hunk 1 above)
Comment (pratclot) [reply TODO.md]: 5d81519b50c2eb5d9f315d4f682aa4c3b81cd434

File: buildtools/.gitattributes (hunk 2 above)
Comment (pratclot) [reply buildtools/.gitattributes:1]: It was auto generated by gradlew when I created the plugin.

**Review by paulacr — APPROVED** [review]

**Pull request merged**